xcode/Subconscious/Shared/Models/Slug.swift

```diff
@@ -37,6 +38,12 @@ public struct Slug:
             options: .regularExpression,
             range: nil
         )
+        .replacingOccurrences(
+            of: "^\(Self.hiddenPrefix)",
+            with: "",
+            options: .regularExpression,
+            range: nil
+        )
```

Comment:
Strip leading underscore in formatting. constructor

Reply:
This will only strip the first leading underscore: To get all of them, we'll need to do something like `^_+`